ci-operator/config/openstack-k8s-operators/swift-operator/openstack-k8s-operators-swift-operator-main.yaml

```diff
@@ -0,0 +1,72 @@
+base_images:
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
+  opm-builder:
+    name: opm-builder
+    namespace: ci
+    tag: latest
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install pre-commit
+  from: src
+  to: openstack-operator-ci-pre-commit
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install python-openstackclient
+    COPY oc /usr/bin/oc
+    COPY opm /usr/bin/opm
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    opm-builder:
+      paths:
+      - destination_dir: .
+        source_path: /bin/opm
+  to: openstack-operator-ci-build-deploy
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: functional
+  commands: export GOFLAGS= && make gotest
```

Comment:
@rabi Some other operators use the below for functional


https://github.com/openshift/release/blob/cbb6ac043dcdfee8677b35fd386750b826b2fe6e/ci-operator/config/openstack-k8s-operators/ovn-operator/openstack-k8s-operators-ovn-operator-main.yaml#L56-L57

~~~
    export GOFLAGS=
    make test GINKGO_ARGS='--no-color'
~~~

Also, In some operators, we are using `gotest`  target and `test` target in others, On further checks I found `gotest` just calls the `test` target so I think that's okay. 

~~~
.PHONY: gotest
gotest: test
~~~

I think disabling ginkgo color output will help for logging, but we can do that in a follow-up.